jax/_src/numpy/ufuncs.py

```diff
@@ -1005,24 +1005,186 @@ def _complex_comparison(lax_op: Callable[[ArrayLike, ArrayLike], Array],
                       lax_op(x.real, y.real))
   return lax_op(x, y)
 
-@implements(np.greater_equal, module='numpy')
 @partial(jit, inline=True)
 def greater_equal(x: ArrayLike, y: ArrayLike, /) -> Array:
+  """Return element-wise truth value of ``x >= y``.
+
+  JAX implementation of :obj:`numpy.greater_equal`.
+
+  Args:
+    x: input array or scalar.
+    y: input array or scalar. ``x`` and ``y`` must either have same shape or be
+      broadcast compatible.
```

Comment:
The doc build error is pointing to this line:
```
/home/runner/work/jax/jax/jax/_src/numpy/ufuncs.py:docstring of jax._src.numpy.ufuncs.greater_equal:9: WARNING: Inline literal start-string without end-string.
```
But I don't see any unmatched inline literal here...

Reply:
Oh, I think I found it, see below.